utils/node_local_test.py

```diff
@@ -300,6 +300,29 @@ def __del__(self):
         if self.running:
             self.stop()
 
+    def _check_timing(self, op, elapsed, max_time):
```

Comment:
I would have probably passed in start rather than elapsed but the end result is the same.

Reply:
That's a good change, just made it.